pkgs/development/php-packages/psysh/default.nix

```diff
@@ -16,7 +16,18 @@ php.buildComposerProject (finalAttrs: {
     url = "https://github.com/bobthecow/psysh/releases/download/v${finalAttrs.version}/composer-v${finalAttrs.version}.lock";
     hash = "sha256-ur6mzla3uXeFL6aEHAPdpxGdvcgzOgTLW/CKPbNqeCg=";
   };
-  vendorHash = "";
+
+  composerRepository = {
```

Comment:
Yeah, this is confusing. I would expect this attribute to be passed the result of `mkcomposerRepository`.

If you apply the suggested change to `buildComposerProjectOverride`, you should be able to use `composerRepository = previousAttrs.composerRepository.overrideAttrs { preBuild = …; };`

Reply:
Yeah I'm not yet at ease with this.... I will do the change.